### ISSUE TITLE
Connect container-external witnesses to the demo

### ISSUE THREAD

**Issue by henkvancann** [peacekeeper/did-webs-iiw-tutorial#10]
I got a few external witnesses offered to test this tutorial with. But I don't have the knowledge (yet) how to do this.
Any suggestions?

**Comment by 2byrds** [comment]
@henkvancann to better understand, do you want to add/remove witnesses that are:

- locally running witness in docker on your machine?
- OR a witness running on the internet?

and see the changes in the did document as you add icp and rot events reflected in the services of the did document?

**Comment by henkvancann** [comment]
Yes, I would like to replace one or more of the locally running witnesses in docker on my machine, with the one offered on that are running on the internet (I think you might know who offered to use them, but I'll PM you about this source :) for this is a public repo).
